src/Files.tsx

```diff
@@ -0,0 +1,15 @@
+import * as Types from './types';
+
+export const Files: React.FC<Types.filesProps> = ({ files }) => {
+  return (
+    <div className="files">
+      <span>+ Files</span>
+      {files.map((file) => <div id={file.id}>
+        {file.title}
```

Comment:
I would prefer this syntax in this case. What do you think?

```suggestion
      {files.map((file) => (
        <div id={file.id}>
          {file.title}
```

Reply:
Seems more like you're indenting it in a similar way to any other loop. It's also nice to return the JSX inside a set of parentheses like that. It seems nice and readable. 

I like it and think we should use it